### ISSUE TITLE
Protecting web access with password

### ISSUE THREAD

**Issue by bluepuma77** [jjethwa/icinga2#40]
To me the documentation is a bit confusing, why I can set a web password and then still the default "password" is used?

> Start a new container and supply the icinga and icinga_web password
> 
> ```
> sudo docker run -e ICINGA_PASSWORD="icinga" -e ICINGA_WEB_PASSWORD="icinga_web" -t jordan/icinga2:latest
> ```
> 
> The Icinga Web interface is accessible at http://localhost/icinga-web with the credentials root:password

In my opinion using default passwords on services that can be externally exposed is dangerous.

It would be great to be able to set username and password via environment variable for
- http://localhost/icinga-web
- http://localhost/icingaweb2

Something along the line of

```
sudo docker run -d --name icinga2 -p 3080:80 -e web-user=user1 -e web-pass=pass1 -e web2-user=user2 -e web2-pass=pass2 jordan/icinga2:latest
```

Maybe stay with the standard user, but still enable to change the default passwords.

**Comment by jjethwa** [comment]
Hi @bluepuma77 

The ICINGA_PASSWORD variable is actually the password for the database, not for the UI.  I should have named it better so add to not cause confusion.  There  is no way to set the UI password at this time.  

There idea is that someone runs the container (with the defaults) then changes the password or uses another authentication module besides file or database.  These changes will persist if the appropriate volumes are used.

So basically, if no volumes are used, it's a default install scenario.  The same thing that would happen if you installed it via a package manager.
